devolutions-session/Cargo.toml

```diff
@@ -35,4 +43,11 @@ features = [
     "Win32_System_IO",
     "Win32_System_Threading",
     "Win32_Security",
+    "Win32_System_Shutdown",
+    "Win32_UI_WindowsAndMessaging",
+    "Win32_UI_Shell",
+    "Win32_System_Console",
+    "Win32_System_Pipes",
+    # Required for Win32_System_Registry
+    "Win32_System_Registry",
```

Comment:
Maybe we can remove this comment 🤣
It would be more interesting to put a comment with information about the codebase that we can't know just by looking at the name of the feature

Reply:
This comment is just wrong xD I remember I wanted to comment about "Win32_System_Registry" requirement for a feature unrelated to the registry.